Services/source/housekeeping/housekeeping_service.c

```diff
@@ -227,9 +228,16 @@ Result mock_everyone(All_systems_housekeeping* all_hk_data) {
 
   //Athena
   uint8_t i;
-  for (i = 0; i < 6; i++) {
+  for (i = 0; i < 2; i++) {
```

Comment:
We ought to have a definition of the number of elements in the temperature array.

Reply:
Done.